src/ai/backend/manager/registry.py

```diff
@@ -1838,7 +1839,11 @@ async def destroy_session(
         async with self.db.begin_readonly() as conn:
             session = await session_getter(db_connection=conn)
         if not reason:
-            reason = "force-terminated" if forced else "user-requested"
+            reason = (
+                KernelLifecycleEventReason.FORCE_TERMINATED
+                if forced
+                else KernelLifecycleEventReason.USER_REQUESTED
+            )
```

Comment:
This `reason` variable is used below to populate `SessionCancelledEvent` and `KernelCancelledEvent`, so please update type definition of `SessionCreationEventArgs` and `KernelCreationEventArgs`'s `reason` variable accordingly, just like I mentioned on review above.

Reply:
Also note that you'll need to change all other codes populating `Session*Event` or `Kernel*Event` event with `reason` variable.

Reply:
Also note that you'll need to change all other codes populating `Session*Event` or `Kernel*Event` event with `reason` variable.